content/shared/v3-enterprise-get-started/_index.md

```diff
@@ -231,7 +231,125 @@ influxdb3 serve --help
 When starting {{% product-name %}} for the first time, it prompts you to enter an email address for verification. You will receive an email with a verification link.
 Upon verification, the license creation, retrieval, and application are automated.
 
-_During the alpha period, licenses are valid until May 7, 2025._
+_During the beta period, licenses are valid until May 7, 2025._
+
+### Authentication and authorization
+
+{{% product-name %}} uses token-based authentication and authorization which is enabled by default when you start the server.
+With authentication enabled, you must provide an _admin token_ or _database token_ to access the server.
+
+- **admin token**: Grants access to all CLI actions and API endpoints.
+_ **database token**: Scoped to a specific database and grant access to write and query data in that database.
+  Creating a database token requires an admin token.
+
+After you have [started the server](#start-influxdb), you can create and manage tokens using the `influxdb3` CLI or the HTTP API.
+
+When you create a token InfluxDB 3 returns a token string in clear text
+that you can use to authenticate CLI commands (using ) and API requests.
+
+
+Securely store your token, as you won't be able to retrieve it later.
```

Comment:
Redundant with the important block below, but I might move that block up to here.
```suggestion
```